src/client/components/app/Transactions/EnableSpigotTx.tsx

```diff
@@ -102,6 +104,15 @@ export const EnableSpigotTx: FC<EnableSpigotTxProps> = (props) => {
     }
   });
 
+  useEffect(() => {
+    console.log('made it to revenue call');
+    if (revenueContractAdd.length === 42) {
```

Comment:
In utils we have a function for this:

util/isValidAddress 

You can use it like this

if (  isValidAddress(revenueContractAddress) ) {
     do whatever
}